package.json

```diff
@@ -23,6 +23,8 @@
     "database"
   ],
   "dependencies": {
+    "@types/nano": "^6.4.5",
+    "@types/request": "^2.47.0",
```

Comment:
I'm not 100% sure on that. But the tests I  have added should lint all the types we are using and check that they are compatible. But I don't think it would catch everything. We can lock those types down to a specific version